### PR TITLE
Execute code resolvers in `fetchFragment` 

### PR DESCRIPTION
**Pull request by oetherington**
Currently, `fetchFragment` only returns resolver-only fields that have SQL resolvers. This PR implements executing the code resolvers for fields without SQL resolvers.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1208173461130215) by [Unito](https://www.unito.io)

### CODE REVIEW

**Comment by oetherington** [comment]
FYI @b0b3rt

**Review by b0b3rt — APPROVED** [review]
Thanks!  lgtm, though I haven't tested it yet.

File: packages/lesswrong/integrationTests/fetchFragment.tests.ts [hunk 1]
```diff
@@ -0,0 +1,84 @@
+import "./integrationTestSetup";
+import { createDummyPost } from "./utils";
+import { fetchFragmentSingle } from "@/server/fetchFragment";
+
+describe("fetchFragment", () => {
```
Comment (s-cheng) [inline packages/lesswrong/integrationTests/fetchFragment.tests.ts:5]: 😍

**Review by s-cheng — APPROVED** [review]
Really appreciate the tests, thanks!

File: packages/lesswrong/server/sql/SelectFragmentQuery.ts [hunk 2]
```diff
@@ -105,20 +104,17 @@ class SelectFragmentQuery<
...
-   * This is commented out for now as it's not needed (due to Apollo filling
-   * in any missing fields) and it has no test coverage.
-   * TODO: Do we actually need this, or can we delete?
-   */
-  /*
   async executeCodeResolvers<T extends DbObject>(
-    obj: T,
+    obj: T | null,
     context: ResolverContext,
-    info: GraphQLResolveInfo,
```
[the remark this answers is not in the document]
Comment (oetherington) [reply packages/lesswrong/server/sql/SelectFragmentQuery.ts:119]: Yeah - the only reason we have this is to pass it on to the resolver functions, but a recent LW PR removed the `info` parameter from resolver functions so it's no longer needed.

**Pull request merged**